allensdk/brain_observatory/sync_utilities/__init__.py

```diff
@@ -13,4 +19,38 @@ def trim_discontiguous_times(times, threshold=100):
     if len(gap_indices) == 0:
         return times
 
-    return times[:gap_indices[0]+1]
\ No newline at end of file
+    return times[:gap_indices[0] + 1]
+
+
+def get_synchronized_frame_times(session_sync_file: Path,
+                                 sync_line_label_keys: Tuple[str, ...]) -> pd.Series:
+    """Get experimental frame times from an experiment session sync file.
+
+    Parameters
+    ----------
+    session_sync_file : Path
+        Path to an ephys session sync file.
+        The sync file contains rising/falling edges from a daq system which
+        indicates when certain events occur (so they can be related to
+        each other).
+    sync_line_label_keys : Tuple[str, ...]
+        Line label keys to get times for. See class attributes of
+        allensdk.brain_observatory.sync_dataset.Dataset for a listing of
+        possible keys.
+
+    Returns
+    -------
+    pd.Series
+        An array of times when frames for the eye tracking camera were acquired.
+    """
+    sync_dataset = Dataset(str(session_sync_file))
```

Comment:
Is there a reason session_sync_file must be a Path, and Dataset only excepts strings?

Reply:
Unfortunately the `Dataset` is kind of in need for a refactor. I'm hoping we can use `Path` going forward, but convert things to `str` because the `Dataset` class is not guaranteed to support it.